fimsauthz-api/fims-authz.yaml

```diff
@@ -291,7 +290,23 @@ components:
             For this server, this is satisfied by the use of a UUID.
         scope:
           description: See securityDefinitions section in this spec.
+          type: array
```

Comment:
scope is being returned as an array of a single string

Reply:
UPP2 implementation also incorporates this

Reply:
It's fine given that the min/max = 1.  I guess this leaves open a policy change later that allows multiple scopes with less impact on clients.